frontend/src/views/MainView.vue

```diff
@@ -67,11 +67,21 @@ export default {
   methods: {
     userSignedIn(resp) {
       this.userData = resp.data
-      if (resp.created) {
+      if (resp.created || this.setupIncomplete()) {
```

Comment:
I'm seeing the MainView `doesnt open setupModal when created is false` test failing with this new addition